nautobot/dcim/templates/dcim/controllerdevicegroup_create.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+{% load helpers %}
+{% load static %}
+
+{% block javascript %}
+<script type="text/javascript">
+    const parentInput = $('select#id_parent');
+    const controllerInput = $('select#id_controller');
+
+    const controllerUrl = '{% url 'dcim-api:controllerdevicegroup-list' %}';
+
+    const updateControllerFromParent = (parentUuid) => {
+        $.ajax({
+            url: `${controllerUrl}${parentUuid}/`,
+            method: "GET",
+            success: (data) => {
+                if (data.controller) {
+                    controllerInput.val(data.controller.id);
+                }
+            },
+            error: (error) => console.log("ERROR", error),
```

Comment:
Is it necessary to better handle ajax errors? If so, is there some example available pls.? @glennmatthews

Reply:
I don't think we have an established standard here but maybe should consider doing an `alert()` instead of a `console.log()` for now? No strong opinion.

Reply:
Keeping as is